### PR TITLE
[Merged by Bors] - Update docs w/ private package publishing

### PR DESCRIPTION
**Pull request by digikata**

### CODE REVIEW

File: content/cli/smartmodules/hub.md [hunk 1]
```diff
@@ -73,9 +76,18 @@ infinyon/regex-filter2@0.1.0
 
 SmartModule Packages are listed in the hub as in the form of `<group>/<package name>@version`. 
 
-Group `infinyon` packages are published by InfinyOn, the maker of Fluvio.  Third-party developers may also upload public packages which appear in other groups. The Hub service enforces signing and publishing restrictions on uploaded packages. Once a signed package of given group has been published, only the original signing key will be accepted for other packages in the same group.
+Group `infinyon` packages are published by InfinyOn, the maker of Fluvio.  Third-party developers may also upload public packages which appear in other groups. The Hub service enforces signing and publishing restrictions on uploaded packages. Once a signed package of given group has been published, only the original owner of that package will be accepted into the same group.
```
Comment (ajhunyady) [inline content/cli/smartmodules/hub.md:79]: I don't quite understand this sentence:

```
 Once a signed package of given group has been published, only the original owner of that package will be accepted into the same group.
```

File: content/cli/smartmodules/hub.md [hunk 2]
```diff
@@ -54,16 +54,19 @@ The command **fluvio hub list** will list SmartModules and versions available fo
 
 {{% inline-embed file="embeds/cli/help/fluvio-hub-list.md" %}}
 
-The list command shows Smartmodules by **group/package@version**. Packages uploaded by InfinyOn will be in the
-infinyon group.
+The list command shows Smartmodules by **group/package@version** in the SMARTMODULE column. The Visibility column describes if the package is accessible to
+to the public or private.  If package visibility is **private** only your user login can list or download the package. Packages uploaded by InfinyOn will be in the infinyon group.
```
Comment (ajhunyady) [inline content/cli/smartmodules/hub.md:58]: `or private` -> `or only to the group`.

**Review by ajhunyady — CHANGES_REQUESTED** [review]
Please update the rest of the examples as well.

**Review by ajhunyady — APPROVED** [review]

File: content/cli/smartmodules/hub.md (hunk 1 above)
Comment (ajhunyady) [reply content/cli/smartmodules/hub.md:79]: @digikata I approved, then I noticed that some of the items are still unresolved.

Comment (digikata) [reply content/cli/smartmodules/hub.md:79]: No problem, the final approval is still blocked on the comments.  I'll update for the commands and you'll have a chance to review again

File: content/cli/smartmodules/hub.md (hunk 2 above)
Comment (digikata) [reply content/cli/smartmodules/hub.md]: right now the hub only supports one group member, the owner

File: content/cli/smartmodules/hub.md (hunk 1 above)
Comment (digikata) [reply content/cli/smartmodules/hub.md]: updated!

**Review by tjtelan — APPROVED** [review]

**Pull request closed without merging**